chumweb/www/views/layouts/base.html

```diff
@@ -21,8 +21,8 @@
     </head>
     <body>
         <header class="section-site-info section-secondary">
-            <img src="{{ "static/img/sailfishos-chum.svg" | to_public_url}}" width="84" height="84" alt="Chum icon" />
-            <h1>Chum</h1>
+            <img src="{{ "static/img/sailfishos-chum.svg" | to_public_url}}" width="108" height="108" alt="SailfishOS:Chum icon" />
+            <h4>SailfishOS:Chum</h4>
```

Comment:
Could you perhaps change this to
```html
<h1 style="font-size: 1.25em;">SailfishOS:Chum</h1>
```

And it'll look like this
![image](https://github.com/sailfishos-chum/sailfishos-chum.github.io/assets/4262067/ac382a4f-a380-4b0c-b9df-54e6b6ca5e6c)

I assume you changed it to a `<h4>` to make it smaller, but it breaks the CSS stylesheet a bit because I did not anticipate `<h4>` tags in a `<header>` element. Furthermore, since it is somewhat the website title, I´d like to keep it an `<h1>` for semantic reasons.


Then it'll look like

Reply:
> I assume you changed it to a <h4> to make it smaller, 

Exactly.

> but it breaks the CSS stylesheet a bit because I did not anticipate <h4> tags in a <header> element.

IMO it is looking good, because the separator at the bottom of the PNG and the whole column at the left of the web-page is much wider: the width of the PNG.
Basically this is how I imagined it, I only wish to have made the icon even lager (e.g. 128x128 px<sup>2</sup>)

> Furthermore, since it is somewhat the website title, I´d like to keep it an `<h1>` for semantic reasons.

But there already is a `<h1>` title of this web-page: **Chum Web**
Actually, having two `<h1>` titles on the same web-page is one reason why I suggest to change this.  IMO, generally there should only be a single `<h1>` heading: the page title

But the major reason is that this community repository is indeed called "SailfishOS:Chum" and that is what the logo represents!

P.S.: I also considered to use simple bold text (`<b>`) for the first line under the logo, which results in a bit smaller font size.  I would like to keep it `<h4>`, but if you prefer simple bold text, that is O.K. for me.  I am only in strong opposition to a sole "**Chum**" under the logo (then you might as well put "**Bait**" there :wink:).

Reply:
> IMO it is looking good, because the separator at the bottom of the PNG and the whole column at the left of the web-page is much wider: the width of the PNG above.
Basically this looks (is rendered) how I imagined it, I only wish to have made the icon even lager (e.g. 128x128 px2): may I?
Feel free to make it larger. I believe I just chose the standard Sailfish OS size, did not think it through at all.

> P.S.: I also considered to use simple bold text (`<b>`) for the first line under the logo, which results in a bit smaller font size. I would like to keep it `<h4>`, but if you prefer simple bold text, that is O.K. for me. I am only in strong opposition to a sole "Chum" under the logo (then you might as well put "Bait" there 😉).

The `<h4>` is fine. I believe I removed the margin on top and bottom with `<h1> and <h2> in a header, but that had to do with underlines that one can see in some parts of the sites. Some margin between the logo, site title and subtext indeed looks better. I made a mess of the CSS by making it too complex with all the selectors :)

Reply:
>  if you want to make the logo larger, please do so and then I'll merge this.

Done: 128x128 px<sup>2</sup>

Please squash-merge, these three commits really do not need to be kept separated.

P.S.: Alternatively I can squash-merge this, if you prefer that.